WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/WatchDogView.java

```diff
@@ -176,6 +184,27 @@ public void widgetDefaultSelected(SelectionEvent e) {
 		}, "Refresh.", "");
 	}
 
+	private void createIntervalsList() {
+		UIUtils.createLabel("Show statistics of the past ", intervalSelection);
+		UIUtils.createComboList(intervalSelection, new SelectionListener() {
+
+			@Override
+			public void widgetSelected(SelectionEvent e) {
+				// Selecting an item in the list initiate the change of
+				// corresponding selectedInterval
+				Combo widget = (Combo) e.getSource();
+				selectedInterval = StatisticsInterval.values()[widget
+						.getSelectionIndex()];
+			}
+
+			@Override
+			public void widgetDefaultSelected(SelectionEvent e) {
+			}
+
+		}, new String[] { "10 minutes.", "30 minutes.", "1 hour.", "2 hours.",
```

Comment:
Why not supply these values to the enum IntervalStatistics and then call IntervalStatistics.values()?